Server-Protocol-Buffer/src/main/proto/internal/evreg/event.proto

```diff
@@ -4,9 +4,22 @@ package de.rki.coronawarnapp.server.protocols.internal.evreg;
 
 
 message Event {
+  // 16 random bytes
   bytes guid = 1;
```

Comment:
I don't know why the guid is there in the Event. It is generated by the server, so we don't need to provide it here.

Reply:
GUID is generate on the client , server is verify that it is unique. https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-concept.md#concept-for-event-creation

Reply:
There was this change recently, that says that its generated on the server: https://github.com/corona-warn-app/cwa-app-tech-spec/pull/30/commits/15a0d0e5c8329d2a295de87881dbf119e11088bd

Reply:
If it has to be generated from the server, I suggest to provide no way of setting it in the client as currently it can be set from the client accidentally:
```kotlin
            TraceLocationOuterClass.TraceLocation
                .newBuilder()
                .setGuid(UUID.randomUUID().toString())
                .build()
```

Reply:
Since (according to the tech spec) the client sends this same object as the one in the Server Response, it needs to be there.